flags/gcov.cmake

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2013, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_FLAGS_GCOV_CMAKE)
+  return()
+else()
+  set(POLLY_FLAGS_GCOV_CMAKE 1)
+endif()
+
+include(polly_add_cache_flag)
+
+polly_add_cache_flag(CMAKE_CXX_FLAGS "-fprofile-arcs -ftest-coverage -O0")
```

Comment:
About `-O0`.

Note that `CMAKE_CXX_FLAGS_RELEASE` set to `-O3 -DNDEBUG` by default. It means that result for `Release` build for current version will be:

```
g++ -O0 -ftest-coverage -fprofile-args -O3 *.cpp
```

I'm not sure about priority (I think g++ will use `-O3`) but for the clearness you should set `CMAKE_CXX_FLAGS` to empty string or to `-DNDEBUG`:

```cmake
set(
    CMAKE_CXX_FLAGS_RELEASE
    "-DNDEBUG"
    CACHE
    STRING
    "C++ compiler flags"
    FORCE
)
```